MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1819,6 +1821,920 @@ If the requested :code:`timeout_nsec` exceeds
 :code:`YDB_MAX_TIME_NSEC`, the function returns
 :code:`YDB_ERR_TIME2LONG`; otherwise it returns :code:`YDB_OK`.
 
+=================
+Programming in Go
+=================
+
+Programming YottaDB in the `Go language <https://golang.org/>`_ is
+accomplished through a wrapper that uses `cgo
+<https://golang.org/cmd/cgo/>`_ to provide the Go programmer with a
+YottaDB package. YottaDB C functions are wrapped by Go methods where a
+method can meaningfully be associated with a Go structure, and Go
+functions otherwise.
```

Comment:
".... and otherwise associated with a Go function"?

Reply:
Changed to “and by Go functions otherwise” to tie the phrase to “wrapped”.